docs/mixins/state.md

```diff
@@ -1,41 +1,76 @@
-# Marionette.Toolkit.StateClass
+# StateMixin
 
-`Marionette.Toolkit.StateClass` is a `Marionette.Object` with a `Backbone.Model` for keeping state.
+`StateMixin` is a public mixin for [`App`](./../app.md) or any Marionette class definition that uses a `Backbone.Model` for keeping state.
```

Comment:
That makes sense...I started doing that in some places...but forgot to make the change everywhere.